javascript/js-array-methods/js-array-methods_map/index.js

```diff
@@ -24,10 +24,47 @@ const cards = [
   },
 ];
 
-const lowerCaseAnswers = null; // ['as often as you like.', ...]
+// const lowerCaseAnswers = null; // ['as often as you like.', ...]
+const lowerCaseAnswers = cards.map((card) => {
+  return card.answer.toLowerCase();
+});
```

Comment:
This is totally fine too but a more succinct version could be:
```suggestion
const lowerCaseAnswers = cards.map(card => card.answer.toLowerCase());
```